From 7d2ac13e5a0440f1ee1d86d066afae4295a7a826 Mon Sep 17 00:00:00 2001
From: Philippe Normand <philn@igalia.com>
Date: Mon, 25 Feb 2019 17:00:10 +0000
Subject: [PATCH] wsfix

---
 src/ws.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ws.cpp b/src/ws.cpp
index ba58c98..102c7c4 100644
--- a/src/ws.cpp
+++ b/src/ws.cpp
@@ -30,6 +30,7 @@
 #include "linux-dmabuf/linux-dmabuf.h"
 #include "bridge/wpe-bridge-server-protocol.h"
 #include <cassert>
+#include <string.h>
 #include <sys/socket.h>
 #include <unistd.h>
 
-- 
2.20.1

